Content.Server/Silicons/Borgs/BorgSystem.cs

```diff
@@ -344,6 +374,26 @@ private void UpdateBatteryAlert(Entity<BorgChassisComponent> ent, PowerCellSlotC
     /// </summary>
     public void BorgActivate(EntityUid uid, BorgChassisComponent component)
     {
+        // Goobstation set pOrg name if a pAI wakes up inside it
+        if (_container.TryGetContainer(uid, component.BrainContainerId, out var brainContainer))
```

Comment:
inshallah reverse some of these if statements.
do
```suggestion
        if (!_container.TryGetContainer(uid, component.BrainContainerId, out var brainContainer))
           return;
```